doc/onboarding-extras.md

```diff
@@ -41,6 +41,7 @@
 | upgrading libuv                          | @saghul                                                               |
 | upgrading npm                            | @fishrock123, @MylesBorins                                            |
 | upgrading V8                             | @nodejs/v8, @nodejs/post-mortem                                       |
+| Embedded use or delivery of Node.js      | @nodejs/delivery_channels                                             |
```

Comment:
Can we make sure that `@nodejs/version-management` becomes a child team of `delivery_channels`?

Reply:
Nit: definitely not a fan of using `_` when all our other teams use `-` to separate words. Let's not make people guess!

Reply:
(We use `_` where it's what a module is using, but otherwise, we use `-` as far as I can tell. So this should be `-`.)

Reply:
@ljharb will look to make it a subteam of the new team.  Not exactly sure what that will take but I'll figure it out.